### PR TITLE
feat(core): getUserId

### PR DESCRIPTION
**Pull request by DavidBauduin**

### CODE REVIEW

File: packages/drivekit_core/flutter_drivekit_core_platform_interface/lib/flutter_drivekit_core_platform_interface.dart [hunk 1]
```diff
@@ -37,13 +37,16 @@ abstract class DrivekitCorePlatform extends PlatformInterface {
...
+
   /// Resets the DriveKit SDK.
   Future<void> reset();
 
   /// Check the validity of the generated token
   /// after connecting with API Key and User ID
   Future<bool> isTokenValid();
 
-  /// Deletes a driver's account
+  /// Deletes a driver's account.
```
Comment (giboin) [inline packages/drivekit_core/flutter_drivekit_core_platform_interface/lib/flutter_drivekit_core_platform_interface.dart:50]: Just a small remark, if you could put this type of change in a separate commit that would make the review easier 😁

**Review by giboin — APPROVED** [review]

File: packages/drivekit_core/flutter_drivekit_core_platform_interface/lib/flutter_drivekit_core_platform_interface.dart (hunk 1 above)
Comment (DavidBauduin) [reply packages/drivekit_core/flutter_drivekit_core_platform_interface/lib/flutter_drivekit_core_platform_interface.dart:50]: Ok, sorry! I'll do it the next time :)
